ansible/roles/delfin-installer/scenarios/rabbitmq.yml

```diff
@@ -23,34 +23,34 @@
 
 - name: Add RabbitMQ official repo
```

Comment:
Order of this need to be changed, first  name: Import RabbitMQ public key and then follwed by  name: Add RabbitMQ official repo